app/Providers/AuthServiceProvider.php

```diff
@@ -8,18 +8,6 @@
 use GuzzleHttp\Client;
 use Lcobucci\JWT\Parsing\Decoder;
```

Comment:
Are we using anything from this JWT package? We should also look to remove anything from composer.json we can.

Reply:
Good catch. I removed 3 more: `guzzle`, `jumbojett`, and `jwt`.